db/models.py

```diff
@@ -50,3 +54,80 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class User(AbstractUser):
```

Comment:
The `User` model extends `AbstractUser`, which is a good practice for customizing the default user model in Django.